CMakeLists.txt

```diff
@@ -25,3 +20,31 @@ else()
     target_compile_options(flagser_pybind PUBLIC $<$<CONFIG:RELEASE>: -Ofast>)
     target_compile_options(flagser_pybind PUBLIC $<$<CONFIG:DEBUG>: -O2 -ggdb -D_GLIBCXX_DEBUG>)
 endif()
+
+# flagser with USE_COEFFICIENTS
```

Comment:
Is this part similar to the ripser bindings in `giotto-tda`?

Reply:
It is a similar idea, but complilation flags and targets are different.

Reply:
Thanks